utility/importer/chrome_importer.cc

```diff
@@ -139,12 +139,11 @@ bool SetEncryptionKey(const base::FilePath& source_path) {
 
 std::u16string DecryptedCardFromColumn(sql::Statement* s, int column_index) {
   std::u16string credit_card_number;
-  int encrypted_number_len = s->ColumnByteLength(column_index);
-  if (encrypted_number_len) {
-    std::string encrypted_number;
-    encrypted_number.resize(encrypted_number_len);
-    memcpy(&encrypted_number[0], s->ColumnBlob(column_index),
-           encrypted_number_len);
+  base::span<const uint8_t> encrypted_number_blob = s->ColumnBlob(column_index);
```

Comment:
you can do `ColumnBlobAsAstring`, see https://source.chromium.org/chromium/chromium/src/+/main:components/autofill/core/browser/webdata/autofill_table.cc;l=171;drc=612d14c8a9ad56b0f873a7c69b43da34d3ef31d8;bpv=1;bpt=1?q=UnencryptedCardFromColumn

Reply:
Yep, that looks nicer. Fixed in a5661e8d96